packages/gatsby-plugin-benchmark-reporting/src/gatsby-node.js

```diff
@@ -79,15 +79,16 @@ class BenchMeta {
 
     const nodejsVersion = process.version
 
-    const gatsbyCliVersion = execToStr(`gatsby --version`)
+    // This assumes the benchmark is started explicitly from `node_modules/.bin/gatsby`, and not a global install
+    const gatsbyCliVersion = require(`gatsby-cli/package.json`).version
 
     const gatsbyVersion = require(`gatsby/package.json`).version
 
     const sharpVersion = fs.existsSync(`node_modules/sharp/package.json`)
       ? require(`sharp/package.json`).version
       : `none`
 
-    const webpackVersion = execToStr(`node_modules/.bin/webpack --version`)
+    const webpackVersion = require(`webpack`).version
```

Comment:
Should this be `webpack/package.json`?

Reply:
Good catch. I thought so as well (I noticed later) but the run printed the version properly so I just left it.